src/test/java/core/basesyntax/strategy/impl/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.strategy.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class OperationStrategyImplTest {
+    private static OperationStrategy operationStrategy;
+    private static OperationHandler expectedOperationHandler;
+
+    @BeforeEach
+    public void setUp() {
```

Comment:
```suggestion
    @BeforeAll
    public void beforeAll() {
```
Please check similar cases in other classes.